server/models/announcements.ts

```diff
@@ -0,0 +1,24 @@
+import { DataTypes } from 'sequelize';
+import sequelize from '../database/connection';
+import Classes from './classes';
+
+const Announcements = sequelize.define('Announcement', {
+  id: {
+    type: DataTypes.INTEGER,
+    autoIncrement: true,
+    primaryKey: true,
+  },
+  class_id: {
+    type: DataTypes.INTEGER,
+    allowNull: false,
+  },
```

Comment:
This id will be created when you do the models relation, so remove the class_id.